spec/models/course_user_spec.rb

```diff
@@ -140,7 +141,9 @@
         before do
           create :course_experience_points_record, points_awarded: 150, course_user: student
         end
-        it { is_expected.to eq(1) }
+        it 'returns the correct level number' do
```

Comment:
Was thinking whether to test the logic here, or test whether the method `level_for` is called, or just try one common generic case. Settled on the last option eventually.